src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/pending_resumption_handler.cc

```diff
@@ -0,0 +1,229 @@
+#include "vehicle_info_plugin/pending_resumption_handler.h"
+#include "application_manager/event_engine/event_observer.h"
+#include "application_manager/resumption/resumption_data_processor.h"
+#include "application_manager/message_helper.h"
+
+namespace vehicle_info_plugin {
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "PendingResumptionHandler")
+
+PendingResumptionHandler::PendingResumptionHandler(
+    application_manager::ApplicationManager& application_manager)
+    : application_manager::event_engine::EventObserver(
+          application_manager.event_dispatcher())
+    , application_manager_(application_manager) {}
+
+template <class T, class Container>
+bool Contains(const T& val, const Container& container) {
+  return container.find(val) != container.end();
```

Comment:
@mked-luxoft What if `Container` class has no `find` method?

Reply:
@mked-luxoft Better update helpers::in_range() method. It was created specially for such purposes

Reply:
@AByzhynar fixed in 23e9b6736a69f55e45d495d644b00c6437b37002